Dockerfile

```diff
@@ -16,14 +16,14 @@ RUN go mod download
 
 COPY . . 
 
-RUN go install github.com/UCCNetsoc/fail2rest
+RUN go build -ldflags="-w -s" -o /go/bin/fail2rest
 
 RUN go mod vendor
 
 CMD [ "go", "run", "main.go" ]
 
-FROM scratch
+FROM golang:1.13
```

Comment:
Instead of this, could you make it `scratch` again and add `ENV CGO_ENABLED=0` underneath the other ENV in the previous stage? Then it should run without having the final image be the full golang:1.13 image :)

Reply:
Unfortunately, that's not working. I tried a `scratch` image with `ENV CGO_ENABLED=0` and got
```
2020-10-22 17:34:38+03:00 [ERROR] global.go:35:getGlobalBans() Error getting global bans
	error='Binary was compiled with 'CGO_ENABLED=0', go-sqlite3 requires cgo to work. This is a stub'
```

Without `ENV CGO_ENABLED=0` `scratch` image just failing at start with
```
fail2rest    | standard_init_linux.go:211: exec user process caused "no such file or directory"
fail2rest    | standard_init_linux.go:211: exec user process caused "no such file or directory"
```

Reply:
After some experiments with build parameters, I successfully build image from `scratch`, yeah!